brave-unbreak.txt

```diff
@@ -208,8 +208,10 @@
 @@||wallpapershome.com/scripts/ads.js$script
 ! Anti-adblock: haaretz.com
 @@||haaretz.com/htz/js/advertisement.js
-! animeflv.com
+! animeflv
+@@||animeflv.net/js/adsbygoogle.js$script,domain=animeflv.net
 @@||animeflv.com/js/adsbygoogle.js$script,domain=animeflv.com
+|animeflv.net
```

Comment:
Just double checking, is this correct or a typo?  According to https://adblockplus.org/filter-cheatsheet#blocking2, it looks like this needs to include to protocol to be useful, no?

Reply:
My err. was a typo.